echo/README.md

```diff
@@ -1,5 +1,131 @@
 # Echo Example
 
-This example is designed to illustrate a few simple aspects of coding an actor module with wasmCloud. The first shows how to respond to simple HTTP requests, but this sample also shows how to use `serde` for JSON serialization.
+An actor is the smallest unit of deployable, portable compute within the wasmcloud ecosystem.
+
+Actors are small WebAssembly modules that can handle messages delivered to them by the host runtime and can invoke functions on capability providers, provided they have been granted the appropriate privileges.
+
+Check out https://wasmcloud.dev/reference/host-runtime/actors/ for more.
+
+In this example our actor responds to all incoming HTTP requests by returning it as JSON object.
+
+## How To Run
+
+Please follow these steps to build and run your actor:
```

Comment:
Requiring NATS and a local registry is a really, really high friction way to run echo. I suggest we also include a `manifest.yaml` for the echo sample that includes the echo actor, the HTTP server, and performs the linking. None of that would require NATS, docker, or docker-compose.

Reply:
@autodidaddict Unfortunately I don't know how to run via manifest.fyaml, maybe i missed it. Could you point me to related docs or examples.

Reply:
I guess this one here: https://github.com/wasmCloud/examples/blob/main/wasmcloud-chat/ops/allin1-manifest.yaml. Ok, i will fix it.